Makefile

```diff
@@ -7,36 +7,44 @@ lint:
 mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
-test_srcs := $(wildcard tests/test_*.py)
-all_test_srcs := $(addprefix all__, $(test_srcs))
-integration_test_srcs := $(addprefix integration__, $(test_srcs))
-
-test: lint mypy $(test_srcs)
+tests:=$(wildcard tests/test_*.py)
+serial_tests:=tests/test_search.py \
+	          tests/test_indexer.py \
+			  tests/test_subscriptions.py
+parallel_tests:=$(filter-out $(serial_tests),$(tests))
+	
+# Run all standalone tests in parallel
+#
+test: $(tests)
 	coverage combine
 	rm -f .coverage.*
 
-$(test_srcs): %.py :
-	set -o pipefail \
-	&& DSS_TEST_MODE="standalone" coverage run -p --source=dss -m unittest $(DSS_UNITTEST_OPTS) $@  2>&1 \
-	| sed -e "s/^/$$$$ /"
-
-integration_test: lint mypy $(integration_test_srcs)
+# Serialize the standalone tests that start a local Elasticsearch instance in
+# order to prevent more than one such instance at a time.
+#
+safe_test: _serial_test $(parallel_tests)
 	coverage combine
 	rm -f .coverage.*
 
-$(integration_test_srcs): integration__%.py :
-	set -o pipefail \
-	&& DSS_TEST_MODE="integration" coverage run -p --source=dss -m unittest $(DSS_UNITTEST_OPTS) $*.py 2>&1 \
-	| sed -e "s/^/$$$$ /"
+_serial_test:
+	for test in $(serial_tests); do $(MAKE) $$test; done
```

Comment:
You can just do:

$(MAKE) -j1 $(serial_tests)

Reply:
Will do.